client/src/components/v2/charts/LocationTimeChart.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  AttributionControl,
+  CircleMarker,
+  Map,
+  Polyline,
+  ScaleControl,
+  TileLayer,
+} from 'react-leaflet';
+import 'leaflet/dist/leaflet.css';
+import LeafletCenterControl from 'components/LeafletCenterControl';
+import {
+  useSensorData,
+  useMultiSensorTimeSeries,
+} from 'api/v2/sensors';
+import styles from './LocationTimeChart.module.css';
+
+const MHP_WORKSHOP_LOCATION = [-37.908756, 145.13404];
+
+/**
+ * Checks if a given location is valid
+ *
+ * @param {number[]} location Location to check
+ * @returns {?Array.<number>} LatLng array if location is valid. Null if invalid
+ */
+function validateLocation(location) {
+  if (location[0] && location[1]) {
```

Comment:
- We should check if they are greater/less than certain numbers too!
- Should also check that the location input is a tuple